style.css

```diff
@@ -159,6 +163,47 @@ h1:hover {
 
 .nav-ditems {
   display: none;
+  list-style: none;
+  text-decoration: none;
+  color: #edecff;
+}
+
+.nav-ditems.active {
+  text-decoration: none;
+  position: fixed;
+  width: 89%;
+  height: 100vh;
+  left: 0;
+  top: 0;
+  background: #191765;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  color: #edecff;
+  background-image: url("img/shapes-mobile-menu.png");
+  background-repeat: no-repeat;
+  background-size: 992px;
+  margin: 0;
+  padding-top: 12%;
```

Comment:
- [ ] The project works very well, but I saw some differences in styling, the first one being the page overflow. See below
![overflow](https://user-images.githubusercontent.com/53124548/135893082-46cc9154-0a58-4cc0-aa7b-6a3c6f2adf04.png)

- [ ] If you look properly, you will see that the image below has white images on it. you should follow the given design.
![popup](https://user-images.githubusercontent.com/53124548/135893223-e163339a-c835-445f-8877-a9136e738bad.png)

- [ ] The white images look different from the original, you can see from the images below.

Your design
![white-img1](https://user-images.githubusercontent.com/53124548/135893255-a9971a0c-7b8d-4869-82a2-c6207615dd27.png)

Original
![white-img2](https://user-images.githubusercontent.com/53124548/135893285-d6b1fb56-1b28-48dc-b9da-9e8db8bac41f.png)

Reply:
The mentioned aspects need further improvement. Kindly ensure to improve this:
- [ ] The hamburger menu is not is visible on mobile screen mode, and there some page overflow that still exits

**Your current mobile screen (Browser mobile mode):**
![image](https://user-images.githubusercontent.com/12745474/135934551-67311b23-6f00-4887-bc23-2c70cba57a0b.png)

Reply:
![3F48D553-A678-49E9-BC93-9C0D989F51C2](https://user-images.githubusercontent.com/70269507/135936283-1c916f8b-10c4-44d1-bae7-33bfa95bbd4b.png)

But it is working perfectly here!!!